custom branch configs (#753)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md

id: "cloud-configuring-repositories"
---

> ### Configuring Pull Request (PR) template URLs

Reviewer: ```suggestion
## Configure pull request (PR) template URLs
```

</TabItem>
</Tabs>

> ## Configuring Custom Branches on Repositories

Reviewer: I think we can shorten this as this page refers to repos 
```suggestion
## Configure custom branches
```

> By default in Development Environments, dbt Cloud attempts to reference the `main` branch in connected repositories. If you want to use a different default branch name, you can configure dbt Cloud with a custom branch setting. 

Reviewer 2: Is it still the case that the main branch must be named `main` or `master`?

Author: @amychen1776 dbt Cloud doesn't care what the base branch's name is.

Reviewer 2: Ah epic, as long as it's set as the default? That's cool

> For example, the environment below is configured to use the `develop` branch of the connected repository: 

Reviewer: ```suggestion
For example, you can use the `develop` branch of a connected repository. Edit an environment, then in "General settings" select  **Only run on a custom branch** , and in "Custom branch" type **develop** or the name of your custom branch.
```

```
https://dev.azure.com/<org>/<repo>/_git/dbt/pullrequestcreate?sourceRef={{source}}&targetRef={{destination}}
```

> ### Configuring Custom Branches on Repositories

Reviewer 3: If you make this `##` then the table of contents is going to look more sensible. Right now, this is rendering under the `Example templates` section.

I might actually recommend moving it up above the `Example templates` section with an `h2` altogether?
